c2corg_ui/tests/views/data/area.json

```diff
@@ -0,0 +1,22 @@
+{
+	"quality": "medium",
```

Comment:
That's tabs. As in other json data files, see https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/tests/views/data/outing.json